dags/covid_etl_dag.py

```diff
@@ -110,6 +132,24 @@ def upload_to_snowflake(task_id):
             )
 
             return create_insert_task
+        
+        def qa_checks(**context):
+            try:
+                with open(qa_file, 'r') as fd:
```

Comment:
instead of exception, could we use `os.path.exists`?

Reply:
Yes.